CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
 
 _None_
 
+### Deprecations
+
+* The Swift 4 templates are now deprecated. This means we will no longer test if the generated output is valid Swift code. We will still try to keep these up-to-date with context changes.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/SwiftGen/SwiftGen/blob/16647a8485b64d443c7b75b44d0c30d4215afea5/CHANGELOG.md#L13"></span>Line describing your entry should end with a period and 2 spaces.</td>
    </tr>
  </tbody>
</table>